.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: "[Type] Bug"
+assignees: ''
+
+---
+
+<!-- Thanks for contributing to Sensei! Pick a clear title ("Lesson: Show complexity in individual lessons") and proceed. -->
+
+#### Steps to Reproduce
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+#### What I Expected
+
+
+#### What Happened Instead
+
+
+#### PHP / WordPress / Sensei version
```

Comment:
Shouldn't this be "Sensei Share Your Grade version" instead of "Sensei version"?

Reply:
Thinking more, maybe ask for both versions would be better. What do you think?

Reply:
Sure, that sounds good.